### ISSUE TITLE
Adding additional tables

### ISSUE THREAD

**Issue by buda56** [neozhu/CleanArchitectureWithBlazorServer#700]
Hi,
   Tried adding an additional table to the project, I am having issues trying to add a migration. Can you please advise the steps to add additional tables to the project database.

I created the new class in Domain\Entities folder and added the DBSet entry to the ApplicationDBContext, then tried adding the migration via the ConnectedServices in the Server.UI project. 

Regards
Peter

**Comment by neozhu** [comment]
Sure, I will take some time to write documentation on how to add a entity class -> migration table

**Comment by buda56** [comment]
Hi,
   Did some more testing on this, the issue is with the new VS2022 tool that allows migrations form the connected services feature of the Server.UI project.
I had set the "Default Project" in the Package Manager Console to "srs\Migrators\Migrators.MSSQL" before running the migration option (see screen shot)
<img width="1025" alt="devenv_Ydi0Cg0TmX" src="https://github.com/user-attachments/assets/7611b3e5-d4ae-427c-ad24-05220cf37b77">
I tried running the migration option and got the following error...
<img width="1280" alt="devenv_YAuw5tUhU4" src="https://github.com/user-attachments/assets/224f9591-1767-41c3-be3a-f1f6ead87def">
I subsequently ran the migration from the Package Manager console (successfully)
<img width="898" alt="devenv_u50Cqc3LAt" src="https://github.com/user-attachments/assets/053d40c2-14d4-4a1a-8f06-c47e3e1413b1">

Regards
Peter

**Comment by neozhu** [comment]
<img width="1916" alt="image" src="https://github.com/user-attachments/assets/2e96f0a2-cd6f-4c8d-b223-a780d7089f0e">
first set default startup project : server.UI project
second select default project: src\Migrators\Database in PMC
last run add-migration cmd

**Comment by buda56** [comment]
Yes that is how I got it to work using the PMC, but if you try it using the Connected Services option it fails with the error described above.
<img width="1280" alt="devenv_HfYFiqOI4f" src="https://github.com/user-attachments/assets/085fd4df-63ab-47b4-b365-4925e68f64b1">

Regards
Peter

**Comment by neozhu** [comment]
https://www.youtube.com/watch?v=X1b4hFLs4vo
